kubernetes/akvorado/templates/akvorado-orchestrator-deployment.yaml

```
@@ -0,0 +1,69 @@
+            - -c
+            - |
+              until nc -zvw5 {{ .Release.Name }}-kafka-external-bootstrap.{{ .Release.Namespace }}.svc.cluster.local 9094; do echo "Waiting for Kafka..."; sleep 2; done
+
+      containers:
+        - name: geoip-sidecar
+          image: {{ include "common.images.image" ( dict "imageRoot" .Values.geoip.image ) }}
+          env:
+            - name: IPINFO_TOKEN
+              value: {{ .Values.geoip.env.ipinfo_token | quote }}
+            - name: IPINFO_DATABASES
+              value:  {{ .Values.geoip.env.ipinfo_db | quote }}
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)